roles/config/defaults/main.yml

```diff
@@ -36,12 +36,12 @@ satellite_content_views:
   - name: RHEL7Ansible_CV
     repositories:
       # yamllint disable-line rule:line-length
-      - name: "Red Hat Ansible Engine 2.8 RPMs for Red Hat Enterprise Linux 7 Server x86_64"
+      - name: "Red Hat Ansible Engine 2 RPMs for Red Hat Enterprise Linux 7 Server x86_64"
```

Comment:
I could add a dictionary in the vars, mapping the Satellite version to the Ansible version. Still a bit ugly but less... What do you think @evgeni

Reply:
We had a similar discussion with @ehelms in a different repo and ended up on "make the user enter two things for now", mostly for simplicity's sake.

Not against a dict lookup, just also not thinking it's explicitly required :)

Reply:
Yea, I'm torn. Here, the lifecycle of a Satellite version is longer, and the support is well defined so I would be OK with a dict to specify this and simplify it for a user. Since Satellite version X supports Ansible version Y, we shouldn't allow users to make a choice and get it wrong is where I am leaning.

Reply:
The last added commit defines a default based on the mapping outlined by Evgeni but can be overwritten if the user of the automation "knows better". Ansible "2" is the default if the mapping isn't defined, e.g. for 6.9, which should be fine in most cases.